frontend/src/pages/_app.tsx

```diff
@@ -0,0 +1,25 @@
+import { AppProps } from "next/app";
+import React from "react";
+import { QueryCache, ReactQueryCacheProvider } from "react-query";
+import { ReactQueryDevtools } from "react-query-devtools";
+import CookieBanner from "src/components/CookieBanner";
+import "src/global.scss";
+// (thuang): `layout.css` needs to be imported after `global.scss`
+import "src/layout.css";
+import Layout from "../components/Layout";
+
+const queryCache = new QueryCache();
+
+function App({ Component, pageProps }: AppProps) {
+  return (
```

Comment:
I removed `if (isSSR()) return null;` in the component, because we now do some basic SSR with Nextjs to render the static CSS layout on the server side!

We can further explore SSR to fetch and render more data on the server side down the road for performance + caching optimization. But for now, this will work as a starter!